packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -73,6 +88,8 @@ module.exports = class SingleSpaReactGenerator extends Generator {
     this.orgName = templateOptions.orgName;
     this.projectName = templateOptions.projectName;
 
+    const srcFileExtension = this.typescript ? "tsx" : "js";
```

Comment:
Nice, this is way better than templating the filename itself (what I'd done previously for another generator)